src/openzeppelin/access/ownable.cairo

```diff
@@ -1,9 +1,6 @@
 #[contract]
 mod Ownable {
-    use starknet::ContractAddress;
-    use starknet::ContractAddressZeroable;
-    use starknet::contract_address_const;
-    use starknet::get_caller_address;
+    use starknet::{ContractAddress, contract_address::ContractAddressZeroable, get_caller_address};
```

Comment:
if this is not the linter, i vote for separate lines

Reply:
also couldn't simply be `ContractAddressZeroable`, without `contract_address::`?

Reply:
what do you think @ericnordelo?

Reply:
`ContractAddressZeroable` has been removed from the scope of `starknet`, so we can't use it like that, but the reason they removed it is that we don't need to import it anymore. It is enough just importing the trait instead (`use zeroable::Zeroable;`)

```suggestion
    use starknet::ContractAddress;
    use starknet::get_caller_address;
```